test/main.sh

```diff
@@ -591,7 +591,7 @@ if [ "$#" -gt 0 ]; then
 fi
 
 run_test test_check_deps "checking dependencies"
-run_test test_static_analysis "static analysis"
+# run_test test_static_analysis "static analysis"
```

Comment:
oh f*ck, I had to disable the static check locally because of a broken xgettext. Will update in a sec second :)